src/main/java/games/strategy/engine/lobby/server/userDB/DbUserController.java

```diff
@@ -76,15 +95,14 @@ public void updateUser(final String name, final String email, final String hashe
    * If an error occured, an IllegalStateException will be thrown with a user displayable error message.
    */
   @Override
-  public void createUser(final String name, final String email, final String hashedPassword,
-      final boolean admin) {
-    primaryDao.createUser(name, email, hashedPassword, admin);
+  public void createUser(final DbUser user, final HashedPassword password) {
+    Preconditions.checkArgument(user.isValid(), user.getValidationErrorMessage());
+    primaryDao.createUser(user, password);
     try {
-      // TODO : if there is a timeout on failure, this may need to be done on a background thread
-      implementations.forEach(dao -> dao.createUser(name, email, hashedPassword, admin));
+      new Thread(() -> secondaryDaoSet.forEach(dao -> dao.createUser(user, password)));
```

Comment:
Not such a good eye...  I loaded this branch into Eclipse with static analysis enabled: `The allocated object is never used`. :smiley: